lib/elixir/test/elixir/task_test.exs

```diff
@@ -177,64 +173,70 @@ defmodule TaskTest do
+
       assert {{{:nocatch, :unknown}, _}, {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+               catch_exit(Task.await(task))
     end
 
     test "exits on task error" do
       Process.flag(:trap_exit, true)
       task = Task.async(fn -> raise "oops" end)
-      assert {{%RuntimeError{}, _}, {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+      assert {{%RuntimeError{}, _}, {Task, :await, [^task, 5000]}} = catch_exit(Task.await(task))
     end
 
     test "exits on task undef module error" do
       Process.flag(:trap_exit, true)
       task = Task.async(&:module_does_not_exist.undef/0)
-      assert {{:undef, [{:module_does_not_exist, :undef, _, _} | _]},
-              {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+
+      assert {
+               {:undef, [{:module_does_not_exist, :undef, _, _} | _]},
+               {Task, :await, [^task, 5000]}
+             } = catch_exit(Task.await(task))
     end
 
     test "exits on task undef function error" do
       Process.flag(:trap_exit, true)
       task = Task.async(&TaskTest.undef/0)
-      assert {{:undef, [{TaskTest, :undef, _, _} | _]},
-              {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+
+      assert {{:undef, [{TaskTest, :undef, _, _} | _]}, {Task, :await, [^task, 5000]}} =
+               catch_exit(Task.await(task))
     end
 
     test "exits on task exit" do
       Process.flag(:trap_exit, true)
-      task = Task.async(fn -> exit :unknown end)
-      assert {:unknown, {Task, :await, [^task, 5000]}} =
-             catch_exit(Task.await(task))
+      task = Task.async(fn -> exit(:unknown) end)
+      assert {:unknown, {Task, :await, [^task, 5000]}} = catch_exit(Task.await(task))
     end
 
     test "exits on :noconnection" do
-      ref  = make_ref()
+      ref = make_ref()
       task = %Task{ref: ref, pid: self(), owner: self()}
-      send self(), {:DOWN, ref, :process, self(), :noconnection}
+      send(self(), {:DOWN, ref, :process, self(), :noconnection})
       assert catch_exit(Task.await(task)) |> elem(0) == {:nodedown, :nonode@nohost}
     end
 
     test "exits on :noconnection from named monitor" do
-      ref  = make_ref()
+      ref = make_ref()
       task = %Task{ref: ref, pid: nil, owner: self()}
-      send self(), {:DOWN, ref, :process, {:name, :node}, :noconnection}
+      send(self(), {:DOWN, ref, :process, {:name, :node}, :noconnection})
       assert catch_exit(Task.await(task)) |> elem(0) == {:nodedown, :node}
     end
 
     test "raises when invoked from a non-owner process" do
       task = create_task_in_other_process()
-      message = "task #{inspect task} must be queried from the owner but was queried from #{inspect self()}"
+
+      message =
+        "task #{inspect(task)} must be queried from the owner but was queried from #{
+          inspect(self())
+        }"
```

Comment:
Please split this message in two and concat with `<>` so that we don't get the ugly-looking split on the `#{`.